modules/representation_management/spec/support/rswag_config.rb

```diff
@@ -0,0 +1,158 @@
+# frozen_string_literal: true
+
+class RepresentationManagement::RswagConfig
+  def config # rubocop:disable Metrics/MethodLength
+    {
+      'modules/representation_management/app/swagger/v0/swagger.json' => {
+        openapi: '3.0.1',
+        info: {
+          title: 'va.gov Representation Management API',
+          version: '1.0.0',
```

Comment:
Probably want to use `v0` and `0.1.0` where applicable to be consistent with our namespace